.buildkite/scripts/dra_release.sh

```diff
@@ -42,6 +42,6 @@ mkdir -p ${BASE_DIR}/reports
 cd ${BASE_DIR}/reports && shasum -a 512 dependencies-${VERSION}.csv > dependencies-${VERSION}.csv.sha512
 
 cd $(dirname ${WORKSPACE})
-FOLDER=${FOLDER_PATH}
-OUTPUT_FILE=${DRA_OUTPUT}
+export FOLDER="${FOLDER_PATH}"
+export OUTPUT_FILE="${DRA_OUTPUT}"
```

Comment:
BRANCH, PROJECT, TYPE and VERSION are also used by the script 

Add to those vars also "export"

Reply:
I think the TYPE is the local var, isn't it?

Reply:
That variable in release-manager.sh gets the value from the TYPE variable if it exists
https://github.com/elastic/fleet-server/blob/aeb00c548cd5d3042c9052ef8657e8b677bac101/.buildkite/scripts/release-manager.sh#L20